bundles/CoreBundle/Resources/config/maintenance.yaml

```diff
@@ -8,7 +8,7 @@ services:
             - 'maintenance.pid'
             - '@logger'
             - '@Symfony\Component\Lock\LockFactory'
-            - '@Symfony\Component\Messenger\MessageBusInterface'
+            - '@messenger.bus.pimcore-core'
```

Comment:
How about we use a diff bus for maintenance tasks? Just to make it future-proof, so that we can add specific middlewares for maintenance bus.

Reply:
Like for this use case https://github.com/pimcore/pimcore/pull/10725